supla-common/proto.h

```diff
@@ -529,7 +572,11 @@ extern char sproto_tag[SUPLA_TAG_SIZE];
 #define SUPLA_CHANNEL_FLAG_LIGHTSOURCELIFESPAN_SETTABLE \
   0x02000000  // ver. >= 12
 #define SUPLA_CHANNEL_FLAG_POSSIBLE_SLEEP_MODE_deprecated \
-  0x04000000  // ver. >= 12
+  0x04000000  // ver. >= 12  DEPRECATED
+#define SUPLA_CHANNEL_FLAG_RUNTIME_CHANNEL_CONFIG_UPDATE \
```

Comment:
Why do we need to know if the channel allows you to change parameters in runtime mode, as after registration it already determines whether you can change parameters?

Reply:
This flag informs server that device can accept messages with "set channel config".
Currently we support only "ds_get_channel_config", which is initiated by device itself.
Additionally, this flag means that device can accept configuration without reset of connection. This is important, since many thermostat parameters may be changed online and we don't want to reset connection of all devices/clients in the meantime.